taskDev4/src/textHandler/InvalidSymbols.java

```diff
@@ -0,0 +1,27 @@
+package textHandler;
+
+import java.util.*;
+/**
+ * Describes invalid symbols for calculations.
+ */
+public class InvalidSymbols {
```

Comment:
Actually class does nothing only takes time for some awkward string to list conversion